src/data/queries/newsStory.ts

```diff
@@ -24,10 +19,10 @@ export const params: QueryParams<null> = () => {
 }
 
 // Define the option types for the data loader.
-export type NewsStoryDataOpts = QueryOpts<{
+export type NewsStoryDataOpts = {
   id: string
-  context?: GetServerSidePropsContext
```

Comment:
The `QueryOpts` type definition provided by next-drupal doesn't seem correct to me.

```
export declare type QueryOpts<Options> = Options & {
    context?: GetStaticPathsContext | GetServerSidePropsContext;
};
```
It seems like it should be `context?: GetStaticPropsContext | GetServerSidePropsContext`. After all, we are querying data from within `getStaticProps` and, as such, passing in a `GetStaticPropsContext` property. So, our query functions need to accept a context that is of type `GetStaticPropsContext`.

We have a lot of instances of `QueryOpts<{some type definition}>`, but, for now, I simply removed this from stories and story listings and explicitly added the context property with the type we want (`ExpandedStaticPropsContext`, which is a decorated form of `GetStaticPropsContext`).  We'll need to address the other instances later. Perhaps we need to ask about/contribute to next-drupal re: `QueryOpts` type.